backend/app/processing/db_storage.py

```
@@ -0,0 +1,57 @@
+import os
+import json
+import uuid
+import shutil
+from pathlib import Path
+from typing import List, Dict, Any
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)